### DISCUSSION TITLE
How to make a DAG run backwards

### DISCUSSION THREAD

**Discussion by Taichi10** [apache/airflow#29667] [Q&A]
I want to run a DAG in the backward direction(on the basis of date) how can I do that?
ps I tried to use the run() method and add run_backwards but it was producing some weird result , can anyone help

**Comment by potiuk** [comment]
I don't think it is supported. You likely need to write a custom client to trigger your DAG in reverse direction using the REST API. That would be my best advice. 

And "weird result" of yours is absolutely not actionable. No-one could help you if you write "weird result" without describing what it was (just saying).

**Comment by potiuk** [comment]
BTW. Yoy CAN make a backfill job run backwards if you want to run backfill https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#backfill (see `--backwards` flag). Maybe that's what you were after?

**Reply by Taichi10** [reply]
i want to run DAGs in reverse direction

**Reply by potiuk** [reply]
Let me repeat it: "You CAN make a backfill job run backwards if you want to run backfill https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#backfill (see --backwards flag). Maybe that's what you were after?"
